src/parser/jobs/rpr/modules/DefensiveCooldowns.tsx

```diff
@@ -0,0 +1,10 @@
+import {Defensives} from 'parser/core/modules/Defensives'
+
+export class DefensiveCooldowns extends Defensives {
+	protected override trackedDefensives = [
+		this.data.actions.ARCANE_CREST,
+		this.data.actions.FEINT,
+		this.data.actions.SECOND_WIND,
```

Comment:
Blocking; if stat block isn't good for more than 2, keep the party support options, toss the self-healing for now. it's useful infomation to display.

Reply:
I'm removing this module over tuning it for now. End of the day, the way the module works is counter to what it's meant to convey. If this changes, happy to reintroduce it but I don't wanna force that change since other jobs may have different concerns.